core/src/test/java/org/axonframework/eventsourcing/eventstore/RevisionBasedSnapshotJuryTest.java

```diff
@@ -0,0 +1,70 @@
+package org.axonframework.eventsourcing.eventstore;
+
+import org.axonframework.common.IdentifierFactory;
+import org.axonframework.serialization.AnnotationRevisionResolver;
+import org.axonframework.serialization.Revision;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.Instant;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class RevisionBasedSnapshotJuryTest {
+
```

Comment:
Minor thing, but you've got one unnecessary white space here.

Reply:
Fixed.